tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -871,7 +871,22 @@ def get_model_name_from_args(argv: argparse.Namespace):
     if not isinstance(input_model, (str, pathlib.Path)):
         return output_dir
 
-    input_model_name = os.path.splitext(os.path.split(input_model)[1])[0]
+    # get file name
+    input_model_name = os.path.split(input_model)[1]
+
+    # get base directory name if file name is empty
+    while input_model_name == '' and len(input_model) > 0:
+        input_model_name = os.path.basename(input_model)
+        input_model = os.path.split(input_model)[0]
```

Comment:
can we do this much simpler:

```
input_model_name = os.path.basename(input_model)
if input_model_name == '':
    input_model_name = os.path.basename(os.path.dirname(input_model))
```

Reply:
If input_model is file, for example `../dir/some_model.pb`, then this code will return `dir.xml`
But we need in this case `some_model.xml`.
So I first try to set input_model_name with filename in line 875. And only if it is empty I use directory name.

Reply:
`input_model_name = os.path.basename(input_model)` will return `some_model.pb`. So 875 line is not needed as well.

Reply:
`while` loop looks strange, please believe)

Reply:
Agree. Removed loop.